lib/main.dart

```diff
@@ -152,13 +144,31 @@ class _TenTenOneState extends State<TenTenOneApp> {
     ],
   );
 
-  Future<void> _callInitWallet() async {
-    final appSupportDir = await getApplicationSupportDirectory();
-    await api.initWallet(network: Network.Testnet, path: appSupportDir.path);
+  Future<void> init() async {
+    try {
+      await setupRustLogging();
+
+      final appSupportDir = await getApplicationSupportDirectory();
+      await api.initWallet(network: Network.Testnet, path: appSupportDir.path);
+
+      FLog.info(text: "Starting ldk node");
+      int listeningPort = 9735; // TODO: Move this to settings?
+      api
+          .runLdk(listeningPort: listeningPort)
+          .then((value) => FLog.info(text: "ldk node stopped."))
+          .catchError((error) => FLog.error(text: "ldk stopped with an error", exception: error));
```

Comment:
Do we even have to listen if no one can connect to us?

Reply:
no we don't 🙂